libs/editor/WordPressEditor/build.gradle

```diff
@@ -45,9 +45,9 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:appcompat-v7:23.1.1'
-    compile 'com.android.support:support-v4:23.1.1'
-    compile 'com.android.support:design:23.1.1'
+    compile 'com.android.support:appcompat-v7:23.3.0'
```

Comment:
> For the entire feature, yes. For this PR, no.

Ignore me - I'd forgotten that this will be merged into a feature branch rather than `develop`.